### DISCUSSION TITLE
допоможіть

### DISCUSSION THREAD

**Discussion by holderXCH** [Chia-Network/chia-blockchain#18478] [Support]
доброго дня. 09.08.2024 перевів монетки XCH з біржі OKX вони досі не поступили на гаманець блокчейну Chia . адресу перевірив вона правильна

**Comment by BrandtH22** [comment]
Hey @holderXCH  , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
- If using the cli then I would stop chia, rename the wallet db folder in `~.chia\mainnet\wallet\db` then restart chia.

If the funds are still not visible can you let us know:

- if the wallet icon in the top right is orange or green?
- if orange can you click on the wallet icon and let us know if you see any connected peers?
- can you provide your log file from `~.chia\mainnet\log\debug.log`?

This information has been translated using translation software and might not be entirely accurate

---

Эта информация была переведена с помощью программного обеспечения для перевода и может быть не совсем точной

Привет, @holderXCH, в качестве предупреждения мы, как правило, можем предоставить более своевременную и тщательную поддержку на нашем сервере Discord (https://discord.gg/chia). 

Что я рекомендую попробовать, так это повторную синхронизацию кошелька: 
- Если вы используете графический интерфейс, перейдите в настройки -> advanced -> повторите синхронизацию. Это закроет клиент, а при повторном открытии он снова синхронизируется. 
- Если бы я использовал cli, то я бы остановил chia, переименовал папку wallet db в `~.chia\mainnet\wallet\db`, а затем перезапустил chia. 

Если средства по-прежнему не видны, можете ли вы сообщить нам об этом: 
- Если значок кошелька в правом верхнем углу оранжевый или зеленый? 
- Если оранжевый, вы можете нажать на значок кошелька и сообщить нам, если вы видите какие-либо подключенные пиры?
- Можете ли вы предоставить свой лог-файл из ~.chia\mainnet\log\debug.log?

**Reply by holderXCH** [reply]
![IMG_4909](https://github.com/user-attachments/assets/5f9e35b2-ffee-48b5-a0a4-9954585b1e5e)

**Reply by holderXCH** [reply]
![IMG_4912](https://github.com/user-attachments/assets/80a09607-45d5-4b80-90a6-825155e124a2)

**Reply by holderXCH** [reply]
![IMG_4911](https://github.com/user-attachments/assets/ad331a83-7168-42a7-9d24-d0b2920f6c77)

**Comment by holderXCH** [comment]
Я дуже радий що ви мені відповіли. Я не сильний в програмуванні. Я просто
кожен місяць купляю монети на біржі OKX і виводжу їх на гаманець. Завжди
все було добре. А цього разу вони не прийшли. Значок синхронізації
оранжевий. Я вийшов з гаманця і заново зайшов за допомогою сід фрази. Але
значок знову оранжевий. Що означає лог файл?

   - Можете ли вы предоставить свой лог-файл из
   ~.chia\mainnet\log\debug.log?
   - Я не розумію що це?


Пт, 16 авг. 2024 г. в 20:27, ClydeWallace22 ***@***.***>:

**Reply by BrandtH22** [reply]
Hey @holderXCH , the orange means the wallet is not fully synced. Also it appears you are running a full node which is not fully synced either. 

As a reminder we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

Going through the debug logs will help us determine why those are not syncing fully. Can you open the file explorer and in the address bar enter `%HOMEPATH%/.chia/mainnet/config`, hit ENTER, then upload the file called `debug.log` to this chat?

That will indicate what errors or warnings the chia client is providing that might indicate why it is not syncing currently.

This information has been translated using translation software and might not be entirely accurate

---
Эта информация была переведена с помощью программного обеспечения для перевода и может быть не совсем точной

Привіт, @holderXCH, помаранчевий означає, що гаманець не повністю синхронізований. Крім того, виявляється, що ви використовуєте повний вузол, який також не повністю синхронізований. 

Нагадуємо, що загалом ми можемо надавати більш своєчасну та ретельну підтримку на нашому сервері Discord (https://discord.gg/chia). 

Перегляд журналів налагодження допоможе нам визначити, чому вони не синхронізуються повністю. Чи можете ви відкрити файловий провідник і в адресному рядку ввести `%HOMEPATH%/.chia/mainnet/config`, натиснути ENTER, а потім завантажити файл під назвою `debug.log` у цей чат?

 Це вкаже, які помилки або попередження надає клієнт chia, що може вказати, чому він наразі не синхронізується.

**Comment by Wselennaj** [comment]
Добрый день! У мен такая же  беда . Переводил с биржи и не пришли , кошелёк горит оранжевым. Что делать помогите?

**Reply by BrandtH22** [reply]
Hey @Wselennaj  , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

First make sure you are using the latest version of the chia wallet from here: https://www.chia.net/downloads/

What I recommend trying is a resync of the wallet:

- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
- If using the cli then I would stop chia, rename the wallet db folder in ~.chia\mainnet\wallet\db then restart chia.

If the funds are still not visible can you let us know:

- if the wallet icon in the top right is orange or green?
- if orange can you click on the wallet icon and let us know if you see any connected peers?
- can you provide your log file from ~.chia\mainnet\log\debug.log?

This information has been translated using translation software and might not be entirely accurate

Эта информация была переведена с помощью программного обеспечения для перевода и может быть не совсем точной

Привет, @Wselennaj , в качестве предупреждения мы, как правило, можем предоставить более своевременную и тщательную поддержку на нашем сервере Discord (https://discord.gg/chia).

Во-первых, убедитесь, что вы используете последнюю версию кошелька chia отсюда: https://www.chia.net/downloads/

Что я рекомендую попробовать, так это повторную синхронизацию кошелька:

- Если вы используете графический интерфейс, перейдите в настройки -> advanced -> повторите синхронизацию. Это закроет клиент, а при повторном открытии он снова синхронизируется.
- Если бы я использовал cli, то я бы остановил chia, переименовал папку wallet db в ~.chia\mainnet\wallet\db, а затем перезапустил chia.

Если средства по-прежнему не видны, можете ли вы сообщить нам об этом:

- Если значок кошелька в правом верхнем углу оранжевый или зеленый?
- Если оранжевый, вы можете нажать на значок кошелька и сообщить нам, если вы видите какие-либо подключенные пиры?
- Можете ли вы предоставить свой лог-файл из ~.chia\mainnet\log\debug.log?

**Comment by holderXCH** [comment]
Мені помогло оновлення гаманця Отправлено с iPhone28 авг. 2024 г., в 23:19, Wselennaj ***@***.***> написал(а):﻿
Добрый день! У мен такая же  беда . Переводил с биржи и не пришли , кошелёк горит оранжевым. Что делать помогите?

**Comment by BrandtH22** [comment]
OP indicates the issue is resolved, feel free to reopen this ticket if the issue rearises
